supertokens_python/recipe/session/framework/django/asyncio/__init__.py

```diff
@@ -37,18 +38,21 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[[_T], _T]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
```

Comment:
why is this needed?

Reply:
Usage of `user_context` is 2nd order nested and somehow linter's feel it's not used. 

On removing this line we get:
```
supertokens_python/recipe/session/framework/django/asyncio/__init__.py:39:4: W0613: Unused argument 'user_context' (unused-argument)
```